src/codechecker_script.py

```diff
@@ -126,6 +126,7 @@ def input_data():
 
 def execute(cmd, env=None, codes=[0]):
     """ Execute CodeChecker commands """
+    cmd = sys.executable + ' ' + cmd
```

Comment:
This is actually might be quite dangerous!
I think we should check env requirements for CodeChecker: what is missing in Bazel sandbox?
BTW, there is also {PythonPath} which we may use.